backend/src/main/java/c/team/session/administration/SessionService.java

```diff
@@ -118,4 +119,9 @@ public Session findByGuestApprovalRoomId(UUID guestApprovalRoomId){
         return sessionRepository.findSessionByGuestApprovalRoomId(guestApprovalRoomId)
                 .orElseThrow(() -> new SessionNotFoundException("no session with approval room: " + guestApprovalRoomId));
     }
+
+    public List<Session> findByLeaderAccountId(String leaderAccountId){
+        return sessionRepository.findSessionsByLeaderAccountId(leaderAccountId)
+                .orElse(new ArrayList<>());
```

Comment:
```suggestion
                .orElse(Collections.emptyList());
```

Reply:
O dziena, faktycznie wygląda to znacznie lepiej